index.html

```diff
@@ -97,5 +97,208 @@ <h2 class="more_det">LET’S CONNECT</h2>
         </li>
       </ul>
     </section>
+    <section class="work-card">
+      <div class="work_img"></div>
+      <div class="name-pro">
+        <h3 class="work_name">Tonic</h3>
+      </div>
+      <ul class="work-detail">
+        <li class="canopy">CANOPY</li>
+        <li class="dev">Back End Dev</li>
+        <li class="year">2015</li>
+      </ul>
+      <article class="detail-cls">
+        <p>
+          A daily selection of privately<br />
+          personalized reads; no accounts or<br />
+          sign-ups required.
+        </p>
+      </article>
+      <div class="button-desc">
+        <h4 class="htm">html</h4>
+        <h4 class="css">css</h4>
+        <h4 class="javascript">javaScript</h4>
+      </div>
+      <button class="see-project">See Project</button>
+    </section>
+    <section class="work-card">
+      <div class="work_img2"></div>
+      <div class="name-pro">
+        <h3 class="work_name work-n">
+          Multi-Post<br />
+          Stories
+        </h3>
+      </div>
+      <ul class="work-detail">
+        <li class="canopy">CANOPY</li>
+        <li class="dev">Back End Dev</li>
+        <li class="year">2015</li>
+      </ul>
+      <article class="detail-cls">
+        <p>
+          A daily selection of privately<br />
+          personalized reads; no accounts or<br />
+          sign-ups required.
+        </p>
+      </article>
+      <div class="button-desc">
+        <h4 class="htm">html</h4>
+        <h4 class="css">css</h4>
+        <h4 class="javascript">javaScript</h4>
+      </div>
+      <button class="see-project">See Project</button>
+    </section>
+    <section class="work-card">
```

Comment:
- [ ] Good job implementing the CSS grid system! However, the `display: grid` should be just the main sections wrappers.  The content itself should be aligned with `display: flex`.  

![image](https://user-images.githubusercontent.com/57421823/148490365-1efb51bd-1007-46c4-9c6e-a34d7aae2e4e.png)

That way, the project requirements will be met, and you will master both technologies 💪